### PR TITLE
feat: accept lower version node

### PR DESCRIPTION
**Pull request by portikM**
# Summary

Some of our repos using node 18.16.1 which results in error:
```
The engine "node" is incompatible with this module. Expected version ">=v18.17.1". Got "18.16.1"
```

## PR Checklist

* [ ] Does not introduce dependencies
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -150,7 +150,7 @@
     ]
   },
   "engines": {
-    "node": ">=v18.17.1"
+    "node": ">=v18.16.1"
```
[the remark this answers is not in the document]
Comment (adamdehaven) [reply package.json:153]: If that's the only one I'd say upgrade it. Also, you may be able to provide a broader engines rule in khcp-ui instead (one of these is probably better than ignoring the version)

[the remark this answers is not in the document]
Comment (adamdehaven) [reply package.json:153]: I don't think it's khcp-ui though, I think it's another dependency over there throwing the error

[the remark this answers is not in the document]
Comment (portikM) [reply package.json:153]: So I take we do want both pin node version in this repo and bump node in khcp-ui to `18.17.1`? If so, then what do we want to pin node version here to?

[the remark this answers is not in the document]
Comment (adamdehaven) [reply package.json:153]: No we don't want to pin anything. 

We want to fix forward; find the package that has the engines rule and provided a broader allowed range. All of our repos should be fine on any version 18.x

[the remark this answers is not in the document]
Comment (portikM) [reply package.json:153]: Roger that

**Comment by portikM** [comment]
Opened [this](https://github.com/Kong/khcp-ui/pull/6835) PR to fix-forward instead.

**Pull request closed without merging**